src/main/java/com/bod/MainMenu.java

```diff
@@ -0,0 +1,180 @@
+        optionString = s.nextLine();
+
+        Fare fare = null;
+        int[] servicesArr;
+
+        switch (getParam(optionString))
+        {
+            case SIMPLE:
+                System.out.println("input name, internalCalls(minutes), externalCalls(minutes), smsAmount(number)");
+
+                servicesArr = multipleInput(3);
+
+                fare = new SimpleBasedFare(name, servicesArr[0], servicesArr[1], servicesArr[2]);
+                break;
+            case INTERNET:
+                System.out.println("input name, internalCalls(minutes), externalCalls(minutes), smsAmount(number), megabytes(number)");
+
+                servicesArr = multipleInput(4);
+
+                fare = new InternetBasedFare(name, servicesArr[0], servicesArr[1], servicesArr[2], servicesArr[3]);
+                break;
+            case TV:
+                System.out.println("input name, internalCalls(minutes), externalCalls(minutes), smsAmount(number), tvChannels");
+
+                servicesArr = multipleInput(4);
+
+                fare = new TVBasedFare(name, servicesArr[0], servicesArr[1], servicesArr[2], servicesArr[3]);
+                break;
+            case PREMIUM:
+                System.out.println("input name, internalCalls(minutes), externalCalls(minutes), smsAmount(number), megabytes, tvChannels");
+
+                servicesArr = multipleInput(5);
+
+                fare = new PremiumBasedFare(name, servicesArr[0], servicesArr[1], servicesArr[2], servicesArr[3], servicesArr[4]);
+                break;
+        }
+
+        action.addFare(fare);
+    }
+
+    private void addClient(){
+        System.out.println("Enter name and fare name");
+        String name, fareName;
+
+        name = s.nextLine();
+        fareName = s.nextLine();
+
+        List<Entity> faresList = action.getFaresList();
+
+
+        action.addClient(name, fareName);
+
+    }
+
+    private void outputFaresList(){
+        List<Fare> list = action.getFaresList();
+
+        for(Entity l: list)
+        {
+            System.out.println(l.getName());
+        }
+    }
+
+    private void outputClientsList(){
+
+        List<Client> list = action.getClientsList();
+
+        for(Client l: list)
+        {
+            System.out.println(l.getName());
+        }
+    }
+
+    private void outputClientsNum(){
+        System.out.println(action.getClientsNum());
+    }
+
+    private void sortFares(){
+        List<Fare> sortedList = action.sortFares();
+
+        for(Fare f: sortedList)
+        {
+            System.out.println(f.getName() + " " + f.calculateFee());
+        }
+    }
+
+    private void findFares(){
+        System.out.println("Input fee range(from/to)");
+
+        Double from, to;
+
+        from = s.nextDouble();
+        to = s.nextDouble();
+
+        List<Fare> list = action.getFaresFromRange(from, to);
+
+        for(Fare f: list)
+        {
+            System.out.println(f.getName() + ": " + f.calculateFee());
```

Comment:
formatting

Reply:
REformat code

Reply:
```suggestion
            System.out.println(f.getName() + ": " + f.calculateFee());
```
changed to System.out.printf in last commit